src/components/Form/Form.js

```diff
@@ -20,7 +20,11 @@ const Form = ({ handleAddTip }) => {
 
     const onAddTip = (e) => {
         e.preventDefault();
-        handleAddTip({ title: title, description: description, mod: mod, upvotes: 0, date: Date.now() })
+
+        const date = new Date(Date.now());
```

Comment:
Maybe we could do .tolocalestring() after newDate(Date.now())

Reply:
Done and Done

Reply:
Much better